src/PkgDeps.jl

```diff
@@ -115,18 +115,18 @@ reachable_registries(; depots::Union{String, Vector{String}}=Base.DEPOT_PATH) =
 
 
 """
-    users(uuid::UUID; registries::Array{RegistryInstance}=reachable_registries())
```

Comment:
Is there a reason to remove this from the docstr? Should it not match what the exact signature is?

Reply:
just to keep these lines more readable, i remove the type annotations that aren't used for dispatch. 
There's no loss of info (since it's documented below).
And i did it for consistency.

But also having two methods with similar but different signatures is tricky to document

I think i'd prefer it if we had these two methods with a single docstring: e.g. 
```julia
"""
    users(uuid::UUID; kwargs...)
    users(package_name::String, pkg_registry_name::String="General; kwargs...)
    
# Arguments
- `uuid::UUID`: ...
- `pkg_name::String`: ...
- `pkg_registry_name::String="General"`: ...

# Keywords
- `registries::Array{RegistryInstance}=reachable_registries())`: ...
"""
users(uuid::UUID; kwargs...)
users(package_name::String, pkg_registry_name::String="General; kwargs...)
```
(but because `pkg_registry_name` is a keywords, and exists for only one of the methods, it's a bit tricky to have a single docstring)

Reply:
Yeah the differentiation of arguments and kwargs is a bit confusing, which makes me lean a bit more to having a separate docstr for each of these functions.

Reply:
here's my proposal which hopefully cleans things up a bit #21